numpy/core/tests/test_ufunc.py

```diff
@@ -941,6 +946,27 @@ def test_identityless_reduction_noncontig_unaligned(self):
         a = a[1:, 1:, 1:]
         self.check_identityless_reduction(a)
 
+    def test_initializer_reduction(self):
+        # np.minimum.reduce is an identityless reduction
+
+        # For cases like np.maximum(np.abs(...), identity=0)
```

Comment:
I think this should say `np.max(np.asb(...), initializer=0)`

Reply:
Done!